### DISCUSSION TITLE
Is the price transparency link in the footer required on pages other than the homepage?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/hospital-price-transparency#84] [Q&A] [answered]
There seems to be different descriptions and interpretations of the new requirement to place a link in the footer of the website.
The rule says "including but not limited to the homepage":

[https://www.federalregister.gov/d/2023-24293/p-5886](https://www.federalregister.gov/d/2023-24293/p-5886)
> (ii) A link in the footer on its website, including but not limited to the homepage, that is labeled “Price Transparency” and links directly to the publicly available web page that hosts the link to the machine-readable file.

The Fact Sheet says "a ‘footer’ at the bottom of the hospital’s homepage "

[https://www.cms.gov/newsroom/fact-sheets/hospital-price-transparency-fact-sheet](https://www.cms.gov/newsroom/fact-sheets/hospital-price-transparency-fact-sheet)
> "1) requiring hospitals to place a ‘footer’ at the bottom of the hospital’s homepage that links to the webpage that includes the MRF;"

Many hospitals have added the link to the footer on every page or most pages, but some have only added it on the homepage. This seems to mostly come up for multi-hospital systems where all of the hospitals share a website and each of the hospitals has its own set of pages on the site. The hospital-specific pages don't always have the footer.

Which is the correct interpretation?  Can this be clarified?

Here is an example: [https://www.texashealth.org/Locations/texas-health-rockwall](https://www.texashealth.org/Locations/texas-health-rockwall) _The page for the specific hospital does not have a Price Transparency link in the footer.  If you manage to get to https://www.texashealth.org you can find the link in the footer, but there doesn't seem to be an easy way to navigate to that homepage of the hospital system_

**Comment by CarlsonErin** [comment]
@nmayle Thanks for the comment.  In the final rule we indicated our belief that the addition of standardized hyperlinks in
the footer of hospital websites would aid in the automation of MRF data retrieval by creating a predictable navigation path to internal web pages that describe the HPT program and providing direct links to the MRF location. We further noted that nothing
would preclude a hospital from additionally providing such a link elsewhere on its homepage if the hospital believes it would be necessary for other reasons.  See the discussion at 88 FR 82113.

**Reply by nmayle** [reply]
I didn't intend to address the issue of additional links on the homepage.

Is providing a link in the footer of the homepage of the website and on no other page on the site sufficient?

The fact-sheet language only requires a link on the homepage.  

The rule seems to require a link on "including but not limited to the homepage".  It is hard for me to be sure what the language in the rule means, but the only reasonable interpretation I can come up with is that the footer must be on more than just the homepage.

**Comment by CarlsonErin** [comment, marked as answer]
Is providing a link in the footer of the homepage of the website and on no other page on the site sufficient?      Yes.
